url_shortener_proj/resources/views/details.blade.php

```diff
@@ -0,0 +1,33 @@
+@extends('home')
+@section('content')
+    <div class="col-md-6 col-md-offset-3" style="text-align: center">
+        <br>
+        <div class="col-md-6-col-md-offset-3">
+            <strong>Original Url:</strong>
+            <a href="{{$url->original}}" target="_blank">{{$url->original}}</a>
+
+        </div>
+        <br>
+        <div class="col-md-6-col-md-offset-3">
+            <strong>Generated Short Url:</strong>
+            <a href="{{route('urls.all', $url)}}" target="_blank">{{$url->short}}</a>
```

Comment:
So, what will be the short URL you can use and send to your friends? What part of this page can you copy and paste?

Reply:
Can I fix that with adding prefix "http://"?